apolloschurchapp/index.js

```diff
@@ -1,8 +1,12 @@
 import './loadConfig';
 import { AppRegistry, YellowBox } from 'react-native';
 import ApollosConfig from '@apollosproject/config';
-import Storybook from './storybook';
 
+// temp fix for the promise.finally
+// https://github.com/storybookjs/storybook/issues/8371
+const fn = Promise.prototype.finally;
+const Storybook = require('./storybook').default; // eslint-disable-line
+Promise.prototype.finally = fn; // eslint-disable-line
```

Comment:
what was this about?

Reply:
see comment above: 

https://github.com/storybookjs/storybook/issues/8371